ghost/members-api/lib/repositories/event.js

```diff
@@ -95,9 +95,6 @@ module.exports = class EventRepository {
         if (filters['data.created_at']) {
             options.filter.push(filters['data.created_at'].replace(/data.created_at:/g, 'created_at:'));
         }
-        if (filters['data.source']) {
```

Comment:
Is this definitely not used anywhere else?

Reply:
fair point. this parameter seems like a dynamic one from the filters. best to leave it as is :)